playbooks/byo/openshift-node/scaleup_ami.yml

```diff
@@ -0,0 +1,53 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- name: Ensure there are new_nodes
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tasks:
+  - fail:
+      msg: >
+        Detected no new_nodes in inventory. Please add hosts to the
+        new_nodes host group to add nodes.
+    when:
+    - g_new_node_hosts | default([]) | length == 0
+
+- include: ../../common/openshift-cluster/evaluate_groups.yml
```

Comment:
We should definitely modify std_include.  However, that would be a large refactor touching lots of areas, and it's outside the scope of this patch set.

This is kind of a special case where we're provisioning nodes from an AMI that already has all packages installed.

Reply:
Could you open an issue or card that describes the problem you had when trying to use std_include.yml?  I know there is a lot of refactoring needed there but it would be helpful to start tracking the problems we have so we can prioritize and get them fixed.  Thanks!